scripts/utils/trading_strategy_helpers.js

```diff
@@ -573,27 +573,29 @@ const buildTransferFundsToMaster = async function(masterAddress, withdrawals, li
   const ERC20 = artifacts.require("ERC20Mintable")
 
   // TODO: enforce that there are no overlapping withdrawals
-  const masterTransactions = await Promise.all(withdrawals.map(async withdrawal => {
-    const token = await ERC20.at(withdrawal.tokenAddress)
-    let amount
-    if (limitToMaxWithdrawableAmount) {
-      amount = BN.min(new BN(withdrawal.amount), new BN(await token.balanceOf.call(withdrawal.bracketAddress)))
-    } else {
-      amount = withdrawal.amount
-    }
-    // create transaction for the token
-    const transactionData = await token.contract.methods.transfer(masterAddress, amount.toString()).encodeABI()
-
-    // prepare bracket transaction
-    const transactionToExecute = {
-      operation: CALL,
-      to: token.address,
-      value: 0,
-      data: transactionData,
-    }
-    // build transaction to execute previous transaction through master
-    return buildExecTransaction(masterAddress, withdrawal.bracketAddress, transactionToExecute, artifacts)
-  }))
+  const masterTransactions = await Promise.all(
```

Comment:
I am sorry for these unrelated changes. It is my prettifier. @fedgiac, would it be okay for you, if I am enforcing code prettifing via travis?